dtslint/ts3.5/tslint.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "dtslint/dtslint.json",
+  "rules": {}
+}
```

Comment:
I tried to run `npm test` locally without this file and everything still works fine. If not strictly needed, I'd prefer to remove this

Reply:
discussed offline.
Never set up properly tslint in my VSCode. This file helps in getting an inline preview of failing `$Expect` assertions

Reply:
@giogonzo looks like this file is required by the [ms-vscode.vscode-typescript-tslint-plugin](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin) extension in order to show the failing type-level tests in VSCode

btw I noticed that there's no configured tslint in this project, I'll send another PR for that